com.unity.netcode.gameobjects/Runtime/Messaging/NetworkManagerHooks.cs

```diff
@@ -76,7 +79,7 @@ public bool OnVerifyCanReceive(ulong senderId, Type messageType, FastBufferReade
                 {
                     if (NetworkLog.CurrentLogLevel <= LogLevel.Normal)
                     {
-                        NetworkLog.LogError($"A {nameof(ConnectionRequestMessage)} was received from a client when the connection has already been established. This should not happen. Please report this to the Netcode for GameObjects team at https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/issues and include the following data: Message Size: {messageContent.Length}. Message Content: {NetworkMessageManager.ByteArrayToString(messageContent.ToArray(), 0, messageContent.Length)}");
+                        NetworkLog.LogError($"A {nameof(ConnectionRequestMessage)} was received from a client when the connection has already been established. {transportVersion}. This should not happen. Please report this to the Netcode for GameObjects team at https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/issues and include the following data: Message Size: {messageContent.Length}. Message Content: {NetworkMessageManager.ByteArrayToString(messageContent.ToArray(), 0, messageContent.Length)}");
```

Comment:
There is another thing I would like to see here. If the transport is not UnityTransport, we should not direct users to file bugs with us. We can't fix bugs in other transports so it just clogs our issues list. That language should be conditionally changed to something like "please file a bug with your transport's maintainer" if they aren't using UnityTransport, while keeping it as it is if they are using UnityTransport. Same for all the other cases.

Reply:
Updated the text so it refers the user to report it with the maintainer of the transport layer.